node/src/command.rs

```diff
@@ -42,14 +42,16 @@ fn load_spec(id: &str) -> std::result::Result<Box<dyn sc_service::ChainSpec>, St
 		// Must define the default chain here because `export-genesis-state` command
 		// does not support `--chain` and `--parachain-id` arguments simultaneously.
 		"picasso-dev" => Box::new(chain_spec::picasso_dev()),
-		// westend parachain
+		// Dali (Westend Relay)
```

Comment:
could these to be moved out into enum or constants and documented via /// , kind of clear network topology